front/src/App.test.js

```diff
@@ -1,13 +1,25 @@
 import { render, screen } from "@testing-library/react";
 import { BrowserRouter as Router } from "react-router-dom";
+import reducer from "./store/reducers/reducer";
+import { Provider } from "react-redux";
+import { createStore, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
 import App from "./App";
 
+//this line is for chrome devtools
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+//in the future possibly we would have more reducers, co we would use combineReducers, but for now there is one
+const store = createStore(reducer, composeEnhancers(applyMiddleware(thunk)));
+
```

Comment:
This lines are repeating over and over, probably you need them for all frontend unit tests. If so, please, refactor it to common utils file.
This way future modifications will be easier and you won't have to import all these new dependencies in every test file.